data/records.json

```
@@ -124,5 +124,26 @@
         "company_site": "",
         "license": "freeware",
         "upload_date": "12/11/2021"
-    }
+    },
+    {
+        "name": "IBGEBot",
+        "description_short": "Bot to access the IBGE website and download data in Excel format from the chosen location",
+        "description": "Automation using BotCity FrameWork, which works with computer vision, was developed for the Rocketseat Scholarships challenge at the #DoWhile event. Simple automation to access the IBGE website and download data in Excel format from the chosen location.",
+        "tags": ["WebBot", "web", "IBGE", "Desktop"],
+        "programming_language": "python",
+        "technology": ["botcity"],
+        "OS": ["windows"],
+        "thumbnail_url": "https://imgur.com/jpO9vdK",
+        "youtube_video": "https://youtu.be/dzhSHia9uCA",

```

Here we must use the embedded video URL:
```suggestion
        "youtube_video": "https://www.youtube.com/embed/dzhSHia9uCA",
```